### PR TITLE
Add an warning for using the CNAME parameter

### PR DESCRIPTION
**Pull request by kuc2477**
Added an warning for using the CNAME parameter with custom domain configured Github pages.

See https://github.com/lektor/lektor/issues/322#issuecomment-260652270 for further details.

### CODE REVIEW

File: content/docs/deployment/ghpages/contents.lr [hunk 1]
```diff
@@ -45,14 +45,17 @@ with behavior for GitHub Pages.
 ## CNAME Support
 
 If you want to use a [CNAME :ext](https://en.wikipedia.org/wiki/CNAME) with
-GitHub pages and Lektor you can provide the intended CNAME with the `?cname`
-parameter:
+custom domain configured GitHub pages and Lektor you can provide the intended
```
Comment (krzysztof-sikorski) [inline content/docs/deployment/ghpages/contents.lr]: I am not a native speaker, but this sentence does not sound grammatically correct to me.

Comment (singingwolfboy) [inline content/docs/deployment/ghpages/contents.lr]: I would suggest rewording this paragraph as:

> If you want to use a custom domain with GitHub pages (also known as a [CNAME :ext](https://en.wikipedia.org/wiki/CNAME)), provide the intended CNAME in the target URL using the `?cname` parameter:

File: content/docs/deployment/ghpages/contents.lr [hunk 2]
```diff
@@ -45,14 +45,17 @@ with behavior for GitHub Pages.
...
+custom domain configured GitHub pages and Lektor you can provide the intended 
+CNAME with the `?cname` parameter:
 
 ```ini
 [servers.production]
 target = ghpages://your-user/your-repository?cname=www.example.com
 ```
 
+Note that this will overwrite your Github page's custom domain setting
+with given `?cname` parameter on every deployment.
```
Comment (singingwolfboy) [inline content/docs/deployment/ghpages/contents.lr]: Please change to:

> Note that this will overwrite whatever custom domain you may have set on GitHub with every deployment.

Note that GitHub has a capital H.
